microdnf/commands/advisory/arguments.hpp

```diff
@@ -69,11 +69,10 @@ class AdvisoryUpdatesOption : public libdnf::cli::session::BoolOption {
 };
 
 
-//TODO(amatej): What should be the StringArgumentList used for? just package spec? current dnf also matches AdvisoryIds
 class AdvisorySpecArguments : public libdnf::cli::session::StringArgumentList {
 public:
     explicit AdvisorySpecArguments(libdnf::cli::session::Command & command)
-        : StringArgumentList(command, "package-spec", _("Package spec present in advisories.")) {}
+        : StringArgumentList(command, "advisory-spec", _("Pattern matching advisory names.")) {}
```

Comment:
Looking at this it's a list option, but that's not apparent from the help text.

The description also isn't super clear to me, what about this (not addressing the issue there can be more than one, I'm not entirely sure how that should be presented):
> A pattern matched against advisory names.